t/t0024-content-s3.t

```diff
@@ -139,6 +163,73 @@ test_expect_success HAVE_JQ 'kvs-checkpoint.get foo returns same value' '
         test_cmp value2.exp value2.out
 '
 
+test_expect_success 'config: reload config does not take effect immediately' '
+	flux config reload 2>/dev/null &&
+	flux dmesg | grep content-s3 | tail -1 >reload.log &&
+	grep "changes will not take effect until next flux restart" reload.log
+'
+
+test_expect_success 'save good config' '
+	mv content-s3.toml content-s3.save &&
+	mv creds/creds.toml creds/creds.save
+'
+
+test_expect_success 'config: reload with extra field fails' '
+	cp content-s3.save content-s3.toml &&
+	echo "extrafield = \"failure\"" >>content-s3.toml &&
+	test_must_fail flux config reload
+'
+
+test_expect_success 'config: reload with malformed uri fails' '
+	cp content-s3.save content-s3.toml &&
+	sed -i -e "s/uri =.*$/uri = \"baduri\"/" \
+		content-s3.toml &&
+	test_must_fail flux config reload
+'
+
+test_expect_success 'config: reload with bad credential path fails' '
+	cp content-s3.save content-s3.toml &&
+	sed -i -e "s/credential-file =.*$/credential-file = \"nocreds\"/" \
+		content-s3.toml &&
+	test_must_fail flux config reload
+'
+
+test_expect_success 'config: reload with extra s3 key fails' '
```

Comment:
There is no check in the cred file parsing for an extra key.  This test is failing because the credential file path is still messed up from the previous test.  The test should probably be removed.